client/src/pages/Energisers.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import Navbar from "./components/Navbar";
+import Footer from "./components/Footer.js";
+import EnergisersCard from "./components/EnergisersCard.js";
+import "./Energisers.css";
+// import NumberOfPages from "./components/NumberOfPages";
```

Comment:
Why is there commented out code?

Reply:
I have made a component for NumberOfPages which is some Icon if we needed to go to next page I will add that component and css style to Robert work if it was needed

Reply:
But it's not part of this code